docs/installation.md

```diff
@@ -1,5 +1,63 @@
 # Installation
 
+We recommend you to install r5py and all its dependencies (incl. Java Developent Kit) from conda-forge.  
+The best way to do this is by installing [Mambaforge](https://github.com/conda-forge/miniforge#mambaforge) (available for Windows/Linux/Mac), which is a drop-in replacement to a popular package distribution [Miniconda](https://docs.conda.io/en/latest/miniconda.html),
+but comes with a significantly faster package manager called `mamba`. Mamba solves dependencies and installs Python packages faster than `conda`, 
+and when it is installed from [Mambaforge](https://github.com/conda-forge/miniforge#mambaforge), it by default uses only `conda-forge` channel to fetch 
+packages. This is a good thing because it helps avoiding dependency conflicts and potential issues with licensing (see [this blog post](https://florianwilhelm.info/2021/09/Handling_Anaconda_without_getting_constricted/)).
+If you are new to (sometimes confusing) Python package managers, you can read more information [from here](https://python-gis-book.readthedocs.io/en/develop/part1/chapter-01/nb/05-installation.html).
+
+If you want, you can also install r5py with [conda](#install-using-conda-alternative) and with [pip](#install-using-pip-alternative).
+
+## Install using `mamba` (recommended)
```

Comment:
Connected to this, I thought of uploading a `PKGBUILD` to Archlinux’ AUR, so r5py can be installed via the package manager there. Thinking that it might be smart to also look into supplying `.deb` and `.rpm` packages. All of these could be built automatically and attached to the GitHub release.

Reply:
Yep, we can discuss about these. But I think it would be good to keep in mind, that Linux is still not very commonly used at Universities (although we use it) and most of the people who are likely users of this tool still uses Windows. Hence, to avoid writing different installation instructions for different OS / setups, I think it would be good to install both r5py and openjdk from conda-forge (because this can be done identically, no matter which OS you use). But as you mention, there are some caveats / compromises here between optimal performance and simplicity in installing/documenting the stuff. We can continue discussing these also with broader group who might also have good pointers regarding these.

Reply:
> Connected to this, I thought of uploading a `PKGBUILD` to Archlinux’ AUR, so r5py can be installed via the package manager there. I’m thinking that it might be smart to also look into supplying `.deb` and `.rpm` packages. All of these could be built automatically and attached to the GitHub release.

Sounds great for Linux users!

Reply:
I definitely support listening conda and mamba first, and telling that they are the easier way to install sh*t, especially when on Windows or MacOS.